doc/api/http.md

```diff
@@ -2173,7 +2173,7 @@ changes:
     description: The `options` parameter can be a WHATWG `URL` object.
 -->
 
-* `url` {string | URL}
+* `url` {string | URL} `URL` here indicates an instance of the `URL` class.
```

Comment:
I'd say this is already an improvement over the current documentation but maybe we could try to improve it a bit further? I can't think of a good wording right away though. Maybe @Trott ?

Reply:
Offhand, nothing is leaping to mind. Maybe land as-is and someone might help improve it at a later date?

Reply:
Reminds me of the following bullet point.

https://github.com/nodejs/node/blame/236237829b26c1756104b5a37797a591e8a5aba6/doc/api/esm.md#L1214